manifest.xml

```diff
@@ -1,26 +1,24 @@
 <!-- This is a generated file. It is highly recommended that you DO NOT edit this file. --><iq:manifest xmlns:iq="http://www.garmin.com/xml/connectiq" version="1">
-    <iq:application entry="PowerAdjuster" id="69223114-7b85-11e6-8b77-86f30ca893d3" launcherIcon="LauncherIcon" minSdkVersion="1.2.0" name="PowerAdjuster" type="datafield">
+    <iq:application entry="PowerAdjuster" id="69223114-7b85-11e6-8b77-86f30ca893d3" launcherIcon="LauncherIcon" minSdkVersion="2.2.0" name="PowerAdjuster" type="datafield">
         <iq:products>
             <iq:product id="square_watch"/>
             <iq:product id="tall_watch"/>
             <iq:product id="round_watch"/>
+            <iq:product id="fenix5"/>
+            <iq:product id="fenix5x"/>
+            <iq:product id="fenix5s"/>
             <iq:product id="fr935"/>
             <iq:product id="semi_round_watch"/>
-            <iq:product id="fr630"/>
```

Comment:
I'd rather not delete these, just catch an exception when building antplus objects and treat it as purePower_prop being disabled.